ivy/functional/frontends/numpy/linalg/matrix_and_vector_products.py

```diff
@@ -57,6 +57,13 @@ def einsum(
     return ivy.einsum(subscripts, *operands, out=out)
 
 
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def gradient(a, /):
```

Comment:
The function signature here should match with that of [np.gradient](https://numpy.org/doc/stable/reference/generated/numpy.gradient.html) function.